eventActions/accountabilityActions.js

```diff
@@ -74,17 +96,43 @@ class accountabilityActions {
 			}
 		}
 	}
-
-	// Unpin message by removing the reaction
-	static async userManuallyUnpinsMessage(reaction, user){
-		var curMessage = reaction.message;
-		if(curMessage.channel.id === config.channels.accountability){
-			if(curMessage.author.id === user.id){
-				if(curMessage.pinned){
-					curMessage.unpin();
-					curMessage.channel.send('Hey, ' + user.username + ', I\'ve unpinned your message as requested!');
-				}
+	// Add a random reaction to a message sent
+	static async addReaction(client, message){
+		if(message.channel.id != config.channels.accountability) return;
+		if(message.content.toLowerCase().includes('!unpin')) return;
+		// Define an array of emojis to pull from
+		var random_emotes = config.emotes.accountability_emotes_array;
```

Comment:
Vars and I think you forgot to add the new array to the config file :D

Reply:
Ah right- forgot to add it to example ;-;